corehq/apps/sso/exceptions.py

```diff
@@ -30,3 +30,12 @@ def __init__(self, error, message):
 
     def __str__(self):
         return f"EntraVerificationFailed({self.error}, {self.message})"
+
+
+class EntraUnsupportedType(Exception):
+    def __init__(self, message):
+        super().__init__(f"{message}")
```

Comment:
isn't `message` always a string?

Reply:
Yes! You're right. Fixed in f65a830d233492917160d78562217cb8c0bf4050